src/containers/FilterContainer.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Created by Katarzyna_Bak on 30.05.2017.
+ */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { filterTask, searchTask } from '../actions';
+
+let FilterContainer = ({ dispatch }) => {
```

Comment:
I'd suggest extracting `Filter` component and then only map state and dispatch to it's props. You will get nice separation of logic and display.